src/test/include.am

```diff
@@ -64,7 +64,7 @@ TESTS += src/test/test src/test/test-slow src/test/test-memwipe \
 
 # run a quick test or two
 # this test only uses IPv4
-TEST_CHUTNEY_FLAVOR_QUICK = bridges+hs-v2
+TEST_CHUTNEY_FLAVOR_QUICK = bridges+hs-v23
```

Comment:
Would it make sense for chutney itself to define a flavor called "quick-test" or something, so that Tor doesn't need to know which flavor chutney thinks is best?  Or is that a mistake?

Reply:
No, the best flavour actually depends on the tor version: newer tor versions may support extra features, and use a newer flavour.